roadmap.md

```diff
@@ -20,6 +20,7 @@ Candidate features for future release cycles
 v3.1 - 2020 Q3
  - Low-latency live (LL-HLS, LL-DASH)
  - Own CEA parser, use mux.js only for TS
+ - PlayReady support in HLS
```

Comment:
I don't think this is needed.  I think this doc is describing the things that we're working on.  Since this feature is already added, adding it to this list doesn't seem needed.  We'll add it to the changelog when we make the release.